bot/seasons/evergreen/trivia_quiz.py

```diff
@@ -28,88 +26,87 @@ class TriviaQuiz(commands.Cog):
 
-    @commands.command(name="quiz", aliases=["trivia"])
+    @commands.group(name="quiz", aliases=["trivia"], invoke_without_command=True)
     async def quiz_game(self, ctx: commands.Context, category: str = "general") -> None:
         """
-        Start/Stop a quiz!
+        Start a quiz!
 
-        arguments:
-        option:
-        - start : to start a quiz in a channel
-        - stop  : stop the quiz running in that channel.
+        If the quiz game is running, then the owner or a mod can stop it by using this command
+        without providing any arguments or vice versa.
 
         Questions for the quiz can be selected from the following categories:
         - general : Test your general knowledge. (default)
-        (we wil be adding more later)
+        (More to come!)
         """
-        category = category.lower()
-
         if ctx.channel.id not in self.game_status:
             self.game_status[ctx.channel.id] = False
-            self.player_dict[ctx.channel.id] = {}
 
-        if not self.game_status[ctx.channel.id]:
-            self.game_owners[ctx.channel.id] = ctx.author
-            self.game_status[ctx.channel.id] = True
-            start_embed = discord.Embed(colour=discord.Colour.red())
-            start_embed.title = "Quiz game Starting!!"
-            start_embed.description = "Each game consists of 5 questions.\n"
-            start_embed.description += "**Rules :**\nNo cheating and have fun!"
-            start_embed.set_footer(
-                text="Points for a question reduces by 25 after 10s or after a hint. Total time is 30s per question"
-            )
-            await ctx.send(embed=start_embed)  # send an embed with the rules
-            await asyncio.sleep(1)
+        if ctx.channel.id not in self.game_player_scores:
+            self.game_player_scores[ctx.channel.id] = {}
 
-        else:
-            if (
-                    ctx.author == self.game_owners[ctx.channel.id]
-                    or Roles.moderator in [role.id for role in ctx.author.roles]
-            ):
-                await ctx.send("Quiz is no longer running.")
-                await self.declare_winner(ctx.channel, self.player_dict[ctx.channel.id])
-                self.game_status[ctx.channel.id] = False
-                del self.game_owners[ctx.channel.id]
-            else:
-                await ctx.send(f"{ctx.author.mention}, you are not authorised to stop this game :ghost: !")
+        # Stop game if running.
+        if self.game_status[ctx.channel.id] is True:
+            return await ctx.send(
+                f"Game is already running..."
+                f"do `{self.bot.command_prefix}quiz stop`"
+            )
+            await self.stop_quiz(ctx.author, ctx.channel)
+            return
 
+        category = category.lower()
+        # Send embed showing available categories if inputted category is invalid.
         if category not in self.categories:
             embed = self.category_embed
```

Comment:
It's weird that `category_embed` is a property. Being in-line with `make_start_embed`, I think it's better as a function. Or, you could just make them both attributes set only once in `__init__` since their content is not dynamic as far as I can tell.